src/utils/SiriusAdapter.cpp

```diff
@@ -290,29 +290,10 @@ class TOPPSiriusAdapter :
     // should the ms file be retained (non-converter mode)
     if (!out_ms.empty())
     {  
-      QFile::copy(tmp_ms_file.toQString(), out_ms.toQString());
+      QFile::copy(sirius_tmp.getTmpMsFile().toQString(), out_ms.toQString());
```

Comment:
just realizing: could you also move the file (instead of copying)?
i.e. would the deletion of the (non-existent) `sirius_tmp.getTmpMsFile()` be a problem?

Reply:
I don't think it would be a problem - I can try it out. 

I thought that if debug > 3 - it would be nice to have everything in one place (the directory and the .ms file. If not the tmp_ms will be deleted anyway later on. 

Does that make sense?